lib/Service/MailManager.php

```diff
@@ -23,34 +23,36 @@
 
 namespace OCA\Mail\Service;
 
-use Horde_Imap_Client;
-use Horde_Imap_Client_Exception;
-use Horde_Imap_Client_Exception_NoSupportExtension;
-use Horde_Imap_Client_Socket;
+use OCA\Mail\Db\Tag;
+use OCA\Mail\Folder;
 use OCA\Mail\Account;
-use OCA\Mail\Contracts\IMailManager;
+use Horde_Imap_Client;
+use function array_map;
 use OCA\Mail\Db\Mailbox;
-use OCA\Mail\Db\MailboxMapper;
 use OCA\Mail\Db\Message;
-use OCA\Mail\Db\MessageMapper as DbMessageMapper;
-use OCA\Mail\Events\BeforeMessageDeletedEvent;
+use function array_values;
+use OCA\Mail\Db\TagMapper;
+use Psr\Log\LoggerInterface;
+use Horde_Imap_Client_Socket;
+use OCA\Mail\Db\MailboxMapper;
```

Comment:
can we keep the imports sorted alphabetically?

Reply:
I'm not sure why my IDE is doing that, I'm sorry! I'll see which extension is causing this.